### PR TITLE
Added the possibility of infinite updating of statistics like 'top'

### PR DESCRIPTION
**Pull request by drons**

### CODE REVIEW

**Comment by drons** [comment]
Maybe it will be more appropriate than the #25

File: gpustat.py [hunk 1]
```diff
@@ -492,7 +497,17 @@ def main():
...
     args = parser.parse_args()
 
-    print_gpustat(**vars(args))
+    if args.interval > 0:
+        term = Terminal()
+        with term.fullscreen():
+            while 1:
+                try:
+                    print_gpustat(clear_term=True, **vars(args))
+                    time.sleep(args.interval)
```
Comment (wookayin) [inline gpustat.py]: Why don't we simply clear terminal here, rather than passing parameters everywhere?

**Comment by wookayin** [comment]
Relates #25

File: gpustat.py (hunk 1 above)
Comment (drons) [reply gpustat.py]: Yes, it was in my first commit 800dbf9 . But the data query in some cases takes a long time and the screen starts blinking.

Comment (wookayin) [reply gpustat.py]: I see. It makes sense.

**Comment by Stonesjtu** [comment]
@drons The file structure has changed a lot, do you have the time to solve the conflict.

**Comment by drons** [comment]
Hi! @Stonesjtu 
Maybe you should merge PR to the master branch so that I do not have to use manual merge anymore?
Or do you have any suggestions?

**Comment by Stonesjtu** [comment]
LGTM, I'll spend more time on improving this util this winter.

**Comment by wookayin** [comment]
Sorry for not having been following up on this. The code needs to be further re-structured and polished a bit; let me work by myself or with @Stonesjtu to get this done soon.

**Comment by Stonesjtu** [comment]
I think the PR is good enough to be merged. I'm creating another PR to address the time drifting problem proposed in #36

**Comment by drons** [comment]
I think the time drifting problem is too simple for a dedicated PR)

**Comment by themightyoarfish** [comment]
Will this be merged?

**Comment by Stonesjtu** [comment]
Since it's `top` like behavior, I think we can create another command like `gputop` to run with a built-in screen, while `gpustat`  can be used to print gpu stats repeatedly like `vmstat`.

**Comment by wookayin** [comment]
Blinking still happens because of clearing out the terminal. We should not use `term.clear`. I am refactoring and revising on this to rely on `clear_eol` instead.

**Comment by wookayin** [comment]
Squashed, rebased (because your commit messages were inappropriate) as in #41. Any reviews?

**Comment by drons** [comment]
Good news.

**Comment by wookayin** [comment]
Resolved by #41. Thanks for your contribution!

**Pull request closed without merging**